toml/encoder.py

```diff
@@ -118,6 +118,10 @@ def _dump_float(v):
     return "{}".format(v).replace("e+0", "e+").replace("e-0", "e-")
 
 
+def _dump_int(v):
```

Comment:
Thank you for your pull request!

Could you move `_dump_int` to be inside of `TomlNumpyEncoder`?
I do not think there is any need to modify the existing encoder to extend the Numpy one to support integers.

Reply:
Hi Will,

just to clarify, you are suggesting to revert `int: _dump_int` to `int: lambda v: v` inside the `TomlEncoder` class and to move the `_dump_int` function inside the `TomlNumpyEncode` class?

Cheers.

Reply:
Yes, that is my suggestion exactly :smiley:

Reply:
Done!